src/components/PolicyMasterPanel.js

```diff
@@ -424,6 +424,15 @@ class PolicyMasterPanel extends FormPanel {
                       }
                     />
                   </Grid>
+                  <Grid item xs={3} className={classes.item}>
```

Comment:
@dragos-dobre @olewandowski1 
I have to add backend anyway, but is there any requirement for payer to be required? If yes 'canSave' function should be edited as well.

Reply:
So no changes on the frontend are needed? Or maybe default payer on setting?

Reply:
@wzglinieckisoldevelo Initially, I set the payer as required, but then I saw that the contribution allows for an optional payer, so I changed it back. @dragos-dobre, I'm not sure if I got it right. Should the payer be required or optional?